qiskit/utils/backend_utils.py

```diff
@@ -168,7 +196,10 @@ def is_local_backend(backend):
     Returns:
         bool: True is a local backend
     """
-    return backend.configuration().local
+    backend_interface_version = _get_backend_interface_version(backend)
+    if backend_interface_version <= 1:
+        return backend.configuration().local
+    return False
```

Comment:
(Similar question to `is_simulator_backend` - the version 2+ check seems inaccurate, but we might not care)

Reply:
This is basically the same answer as `is_simulator_backend` , `BackendV2` doesn't have a `local` attribute the same was as `BackendConfiguration`. It's also not really user actionable, like what does a consumer of backend do with this information? It doesn't really change anything to know if it's a local backend vs a remote one. The closest thing in BackendV2 to this is that the `JobV1` class has a provision for async or sync jobs which could be used to influence behavior. 

That being said all this does in `QuantumInstance` is to influence the max # of circuits per job (to fallback to max list size if it's local) because it's making an incorrect assumption that the local backend is basically aer or basic aer and that local backends always accept an arbitrary number of circuits per job. This doesn't make a difference in V2, because the `.max_circuits` attribute is explicitly defined as returning `None` if there is no limit to the number of circuits per job, regardless of whether it's None or not.